components/constellation/constellation.rs

```diff
@@ -1495,6 +1490,7 @@ impl<Message, LTF, STF> Constellation<Message, LTF, STF>
         };
 
         // Create the new pipeline, attached to the parent and push to pending changes
+        self.handle_load_start_msg(load_info.info.new_pipeline_id);
```

Comment:
Yes, there's quite a bit of boilerplate we could DRY. This should be a separate PR though. Not sure anything in the constellation is E-Easy :)